cmd/dmverity-vhd/main.go

```diff
@@ -104,6 +112,11 @@ var createVHDCommand = cli.Command{
 			Usage:    "Required: container image reference",
 			Required: true,
 		},
+		cli.BoolFlag{
```

Comment:
so, instead of adding the flag here. we'll have to add it in the `main` function, just like the `verboseFlag` is being added. To check the global flag, we'll have to call to `ctx.GlobalBool(sourceFlag)` inside the `fetchImageLayers` function.